packages/valory/contracts/gnosis_safe/tests/test_contract.py

```diff
@@ -353,7 +353,10 @@ def _raise_solidity_error(*_: Any) -> None:
                 self.ledger_api, "contract_address", cast(TxData, tx)
             )
             assert "revert_reason" in reason
-            assert reason["revert_reason"] == "ContractLogicError('reason')"
+            assert (
+                reason["revert_reason"] == "ContractLogicError('reason')"
+                or reason["revert_reason"] == "ContractLogicError('reason', None)"
+            )
```

Comment:
@0xArdi currently we have a wide version range for web3py, this error occurs because different versions of wev3py have different string interpolation for the error class. So this will work until we decide to pin a proper version

Reply:
okay makes sense, it was a non-issue anyway